geonode/mngmt_commands_http/decorators.py

```diff
@@ -0,0 +1,31 @@
+#########################################################################
+#
+# Copyright (C) 2021 OSGeo
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+#########################################################################
+class expose_command_over_http:
```

Comment:
I don't know if the idea was to have them exposed by default or not. Better have the @giohappy opinions here

Reply:
The best would be a configuration inside the settings, where we whitelist the commands to be exposed.

On this regard @italogsfernandes I haven't gone through the implementation but I suppose you have configured the authentication and authorizations required to call the commands. Can you tell me how it is managed?

Reply:
Only admin users can call the commands (`permission_classes = [permissions.IsAdminUser]` - [Test Code](https://github.com/italogsfernandes/geonode/blob/cbc33020cabb8d1ebeca11fd3f542bfbd6be433e/geonode/management_commands_http/tests.py#L55:L58)).

Reply:
ok, thx @italogsfernandes. Do you think we can implement the configuration of exposed commands through settings?

Reply:
Yes, I'm working on it.  👨‍💻

Something like the following config inside `settings.py`: 
```python
MANAGEMENT_COMMANDS_EXPOSED_OVER_HTTP = [
    "ping_mngmt_commands_http",
    "importlayers",  # for example...
]
```

Besides "ping_mngmt_commands_http" which other commands would be interesting to be on this list by default? 

[commands.md](https://github.com/GeoNode/geonode/files/7178528/commands.md)

Reply:
I'd say the following:

- `sync_geonode_datasets`
- `sync_geonode_maps`
- `importlayers`
- `set_all_datasets_metadata`

Reply:
@giohappy changed on bb13b3b856a48083045f36ff2d5a8e4ae568f9f0